plugins/board-resources/src/components/popups/CopyCard.svelte

```diff
@@ -65,6 +75,83 @@
       value,
       newCardId
     )
+
+    if (keepAttachments) {
+      const sourceAttachments = await client.findAll(attachment.class.Attachment, {
+        space: object.space,
+        attachedTo: object._id
+      })
+
+      for (let i = 0; i < sourceAttachments.total; i++) {
+        const a = sourceAttachments[i]
+
+        await client.addCollection(
+          attachment.class.Attachment,
+          selected.space,
+          newCardId,
+          board.class.Card,
+          'attachments',
+          {
+            name: a.name,
+            file: a.file,
+            type: a.type,
+            size: a.size,
+            lastModified: a.lastModified
+          }
+        )
+      }
+    }
+
+    if (keepComments) {
+      const sourceComments = await client.findAll(chunter.class.Comment, {
+        space: object.space,
+        attachedTo: object._id
+      })
+
+      for (let i = 0; i < sourceComments.total; i++) {
+        const comment = sourceComments[i]
+
+        const commentAttachments =
```

Comment:
Comments could have attachments by itself, so it it better to have some logic based on model and collection types. instead of individual collections.